src/function/scalar/nested/struct_pack.cpp

```diff
@@ -34,7 +32,7 @@ static void struct_pack_fun(DataChunk &args, ExpressionState &state, Vector &res
 
 static unique_ptr<FunctionData> struct_pack_bind(ClientContext &context, ScalarFunction &bound_function,
                                                  vector<unique_ptr<Expression>> &arguments) {
-	set<string> name_collision_set;
+	std::set<string> name_collision_set;
```

Comment:
Can this be turned into an unordered set, or is set required here? (I believe `set` is implemented as a tree and `unordered_set` as a HT - not very significant but unordered_set is generally a bit more efficient if you only need to do point inserts/queries, and we use it "by default" in most places so it's a bit more consistent).

Reply:
sure did not do so much semantic analysis